tamr_toolbox/project/_common/movement.py

```diff
@@ -0,0 +1,137 @@
+"""Tasks related to project movement as part of Tamr projects"""
+from typing import List, Optional
+import logging
+
+from tamr_unify_client.project.resource import Project
+from tamr_unify_client import Client
+
+from tamr_toolbox import utils
+from tamr_toolbox.utils.operation import Operation
+
+LOGGER = logging.getLogger(__name__)
+
+
+def export_project(
```

Comment:
Adding `project` to the name might not be necessary as you are already within the `tbox.project` namespace.